ts4.1/index.d.ts

```diff
@@ -184,15 +191,15 @@ type NormalizeWithKeyPrefix<
     : Normalize<T[K]>
   : never;
 
-type KeyPrefix<N extends Namespace> =
+export type KeyPrefix<N extends Namespace> =
   | (N extends keyof DefaultResources ? Normalize<DefaultResources[N]> : string)
   | undefined;
 
 export type TFuncKey<
   N extends Namespace = DefaultNamespace,
   TKPrefix = undefined,
   T = DefaultResources
-> = N extends (keyof T)[] | Readonly<(keyof T)[]>
+> = N extends Array<keyof T> | Readonly<Array<keyof T>>
```

Comment:
We are in favour of shorthand syntax `[]`

Reply:
Changed it also only because of TSLint. Adapt as well?
```
ts4.1/index.d.ts:202:19 - Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.
```